src/IO/InterfileHeader.cxx

```diff
@@ -265,7 +266,12 @@ bool InterfileHeader::post_processing()
       catch(...)
         {}
     }
-
+  
+  if (!isotope_name.empty()){
+      this->exam_info_sptr->set_radionuclide(isotope_name);
+      info("isotope name is " + isotope_name);
```

Comment:
```suggestion
```
none of the other parsing keywords provide this feedback. People can get it via `list*_info` etc.